src/element/Textarea.tsx

```diff
@@ -37,29 +39,39 @@ const UserItem = ({ pubkey, display_name, picture, nip05, ...rest }: User) => {
   )
 }
 
-export default class Textarea extends Component {
-  render() {
-    // @ts-expect-error
-    const { users, onChange, ...rest } = this.props
+function normalizeUser({ pubkey, about, nip05, name, display_name }: User) {
+  return { pubkey, about, nip05, name, display_name }
+}
+
+const Textarea = ({ users, onChange, ...rest }: any) => {
+    const normalizedUsers = Object.keys(users).reduce((acc, pk) => {
+      return {...acc, [pk]: normalizeUser(users[pk]) }
+    }, {})
+    const dbUsers = useLiveQuery(
+      () => db.users.toArray().then(usrs => {
```

Comment:
We can query directly inside here instead of cloning the entire cache?

Reply:
Yes I need to do that.